doc-content/enterprise-only/admin-and-config/setup-auto-cleanup-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='setup-auto-cleanup-proc']
+= Setting up automatic cleanup job
+
+You can set up an automatic cleanup job in {CENTRAL}.
+
+.Procedure
+. In {CENTRAL}, go to *Deploy* > *Jobs*.
+. Click *New Job*.
+. Enter a name, due date and time and following into the *Type* text field.
++
+[source]
+----
+org.jbpm.executor.commands.LogCleanupCommand
+----
+
+. Click *Add Parameter* if you wish to use the <<logcleanupcommand_parameters_{context}, parameters>>, enter a parameter in the key section and enter true or false in the value section.
```

Comment:
Not all parameters have true/false as their value. So I would just state here enter the parameter value in the value section or similar.

Reply:
Made this change